pep-9999.rst

```diff
@@ -0,0 +1,293 @@
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 20-Sep-2019
+Python-Version: 3.9
+
+
+Abstract
+========
+
+Rather than proposing a change to CPython's release cadence (as PEP 602 does),
+or a change to CPython's policy for when new features can be added to a release
+series (as PEP 598 does), this PEP proposes that we instead amend CPython's
+pre-release management process as follows:
+
+* feature freeze, ABI freeze, bytecode format freeze, and maintenance branch
+  creation all correspond with the first release candidate for an X.Y.0 release
+* the X.Y.0 release candidate period is extended from 3 weeks to 2 months
+* beta releases occur every other month whenever there is no X.Y.0 release in
+  the release candidate phase
+* during the beta release period, all newly added interfaces are considered
+  provisional, and each new release may introduce new deprecation warnings
+  (or escalate previous deprecation warnings to errors)
+* routine alpha releases are removed from the process (release managers may
+  still choose to publish them if they feel they are appropriate)
+
+As part of this change, the current "not for production use" guidance given for
+beta releases would be amended to say "suitable for production use only in
+environments with sufficiently robust compatibility testing and operational
+monitoring capabilities".
+
+This PEP also proposes that the frequency of X.Y.0 releases be adjusted to
+target publishing a new release series in August of odd-numbered years
+(2021, 2023, etc).
+
+
+Background
+==========
+
+The current CPython pre-release and release management processes were developed
+in an era where automated continuous integration and operational monitoring
+systems were still relatively immature. Since that time, many organisations
+have adopted deployment models that allow them to incorporate new CPython
+feature releases without substantially more risk than they incur for any other
+code change.
+
+In light of that change in the wider environment, PEP 602 has proposed reducing
+the feature delivery latency for the Python standard library and CPython
+reference interpreter by increasing the frequency of CPython feature releases
+from every 18-24 months to instead occur every 12 months.
+
+Unfortunately, for many organisations, the cost of adopting a new Python release
+doesn't automatically scale down with a reduced number of changes in the release,
+as the primary costs aren't associated with resolving any discovered issues:
+the primary costs are associated with the *search* for issues. This search may
+involve manual testing of software systems, human review of written materials,
+and other activities where the time required scales with the size of the
+existing system, rather than the number of changes between the versions of
+Python.
+
+PEP 598 was an initial attempt by one of the authors of this PEP to propose
```

Comment:
Do the PEP reST files support Sphinx roles? If so, it might be helpful to provide an inline link to the other PEPs where appropriate.

```suggestion
:pep:`598` was an initial attempt by one of the authors of this PEP to propose
```

Reply:
No Sphinx roles (yet), but text of the form "PEP NNN" gets converted to a link automatically.